.vscode/settings.json

```diff
@@ -4,5 +4,67 @@
   "editor.codeActionsOnSave": {
     "source.fixAll": "explicit"
   },
-  "typescript.tsdk": "node_modules/typescript/lib"
+  "typescript.tsdk": "node_modules/typescript/lib",
+  "restoreTerminals.runOnStartup": true,
+  "restoreTerminals.keepExistingTerminalsOpen": true,
+  "restoreTerminals.terminals": [
```

Comment:
VSCode settings committed unintentionally?

Reply:
🤦🏻  leftover

Reply:
but it means a wrong .gitIgnore file definition

Reply:
now I see that in `.gitignore` file⬇️  in the past it was versioned intentionally, we could discuss if we want hold this file versioned.
Anyway in my case I have added an extensions to allow a quickly default terminals session when project will be reopened.  
```
!.vscode/settings.json
```